docs/docs/guide/Model.md

```diff
@@ -471,6 +471,7 @@ You can also pass in a `settings` object parameter to define extra settings for
 |------|-------------|------|---------|
 | return | What the function should return. Can be `document`, or `request`. In the event this is set to `request` the request Dynamoose will make to DynamoDB will be returned, and no request to DynamoDB will be made. | String | `document` |
 | condition | This is an optional instance of a Condition for the update. | [dynamoose.Condition](Condition.md) | `null`
+| returnValues | Set which documents to return after the update. Valid values are: `NONE`, `ALL_OLD`, `UPDATED_OLD`, `ALL_NEW` `UPDATED_NEW` | String | `ALL_NEW`
```

Comment:
```suggestion
| returnValues | Set which documents to return after the update. This setting will be passed into the DynamoDB `ReturnValues` parameter. | String | `ALL_NEW`
```

Couple things here. Making it clear how it'll be used in relation to DynamoDB, and removing the valid DynamoDB options. If DynamoDB were to ever update their values, this documentation would be outdated. I tend to like to keep our documentation as focused on Dynamoose as possible. There are a lot of drawbacks to repeating the DynamoDB documentation in my opinion.